src/seaport.ts

```diff
@@ -2526,7 +2567,8 @@ export class OpenSeaPort {
     const { target, calldata, replacementPattern } = encodeBuy(
       schema,
       wyAsset,
-      accountAddress
+      accountAddress,
+      sellOrder ? undefined : merkleValidatorByNetwork[this._networkName]
```

Comment:
doesn't this need to be `sellOrder.isWaitingForCounterOrder`?

Reply:
ah, yeah i removed it to test something but can add it back